src/Microsoft.IdentityModel.JsonWebTokens/JsonWebToken.cs

```diff
@@ -1020,7 +1048,7 @@ public string Actor
         {
             get
             {
-                _act ??= Payload.GetStringValue(JwtRegisteredClaimNames.Actort);
+                _act ??= Payload.GetStringValue(ActorClaimName);
                 return _act;
```

Comment:
I think the comment here may be incorrect, might want to check if the returned value is 'null' or 'string.empty' if the Actor claim is not found.